Dockerfile.scratch

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.14-alpine as builder
+RUN apk add build-base
+WORKDIR /go/src/github.com/adrienkohlbecker/traefik-auth-cloudflare
+COPY . .
+# Static build required so that we can safely copy the binary over.
+RUN go install -ldflags "-linkmode external -extldflags -static" github.com/adrienkohlbecker/traefik-auth-cloudflare
+
+FROM alpine as alpine
+RUN apk --no-cache add tzdata zip ca-certificates
+WORKDIR /usr/share/zoneinfo
+RUN zip -r -0 /zoneinfo.zip .
+RUN adduser --uid 757 -D golang
```

Comment:
Curious, is this uid value significant?

Reply:
No, it's just a number to represent the non-root user we've created, so we can run on e.g. k8s with a pod security policy specifying non-root users, or openshift - these can't introspect our passwd file to work out whether a named user is root or not.